addons/tutorials-button/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Hide Tutorials button",
-  "description": "Hides the Tutorials button in the editor menu bar.",
+  "name": "Hidden \"Tutorials\" button",
+  "description": "Hides the \"Tutorials\" button from the editor menu bar.",
```

Comment:
Seems counterintuitive because in hindsight, it could imply something it just adds a hidden button (and we think like, why?), and not hide the button itself.

Reply:
This also applies to other "hide XYZ" addons.

Reply:
This also applies to other "hide XYZ" addons.